### PR TITLE
(375) "Health needs" task: "Learning difficulties and neurodiversity" page

### PR DESCRIPTION
**Pull request by edavey**
[Trello 375](https://trello.com/c/SmsCqlYJ/375-health-subsection-learning-difficulties-and-neurodiversity)

learning_difficulties

### CODE REVIEW

**Comment by edavey** [comment]
[Health subsection - Learning difficulties and Neurodiversity](https://trello.com/c/SmsCqlYJ/375-health-subsection-learning-difficulties-and-neurodiversity)

**Review by patrickjfl — APPROVED** [review]

File: server/views/applications/pages/health-needs/learning-difficulties.njk [hunk 1]
```diff
@@ -3,8 +3,183 @@
 
 {% block questions %}
 
-  <p class="govuk-body">
-    Learning difficulties page
+  {% set learningNeedsFollowUp %}
+  {{
+      formPageInput(
```
Comment (Harriethw) [inline server/views/applications/pages/health-needs/learning-difficulties.njk:8]: I believe all the `formPageInput`s should be `formPageTextArea`s for this template?

File: server/form-pages/apply/risks-and-needs/health-needs/learningDifficulties.ts [hunk 2]
```diff
@@ -32,11 +78,51 @@ export default class LearningDifficulties implements TaskListPage {
...
+    }
+    if (this.body.hasDifficultyInteracting === 'yes' && !this.body.interactionDetail) {
+      errors.interactionDetail = 'Describe their difficulties interacting'
+    }
+
+    if (!this.body.requiresAdditionalSupport) {
+      errors.requiresAdditionalSupport = `Confirm whether additional support is required`
+    }
+    if (this.body.requiresAdditionalSupport === 'yes' && !this.body.addSupportDetail) {
+      errors.addSupportDetail = 'Describe their additional needs'
```
Comment (Harriethw) [inline server/form-pages/apply/risks-and-needs/health-needs/learningDifficulties.ts:106]: maybe one for Chanel, but should the copy here be something more like 'Describe the type of support required'?

Comment (edavey) [reply server/form-pages/apply/risks-and-needs/health-needs/learningDifficulties.ts:106]: thks, that is better, will tweak.

**Pull request merged**